README.md

```diff
@@ -1,15 +1,16 @@
 # Decomp Jenkins Agent Dockerfile
 
-1. Build the docker image:
-`docker build . -t decomp`
+1. Obtain an agent name and agent secret from your network administrator.
```

Comment:
If we want to re-organize the instructions, I'm cool with that, but I prefer to keep this a lot less general. This repo only pertains to our specific jenkins instance, and I'm the one who sets it up, so if we're gonna name who to get the stuff from, I think we should just say "Obtain an agent name and agent secret from Ethan". If we did want to make it more general, I'd argue it should say Jenkins administrator rather than network administrator